x-pack/test/apm_api_integration/basic/tests/index.ts

```diff
@@ -26,6 +26,7 @@ export default function apmApiIntegrationTests({ loadTestFile }: FtrProviderCont
     // TODO: we should not have a service overview.
     describe('Service overview', function () {
       loadTestFile(require.resolve('./service_overview/error_groups'));
+      loadTestFile(require.resolve('./service_overview/dependencies'));
```

Comment:
I think this test should be placed inside `Services` not `Service overview`, since we don't have `service_overview` API.

Reply:
What do you mean with "we don't have a service_overview API"?

Reply:
Service Overview is the Page, it consumes the `transactions`, `services`, `errors` APIs.

Reply:
The errors APIs are in separate files as well (`server/routes/errors.ts`). Metrics are service-specific as well, but both their tests and API definitions are in separate files. Moreover, the error APIs on the service overview page and the service error group page have different data needs. I agree some clean up is needed - but I think we should consider being _more_ specific instead of less. Thoughts @sqren?

Reply:
I agree that the current API definition and where it's placed is a bit of a mess right now, but I think from an API point of view there is no `Service overview`. What I understood is `Service Overview` is a page which shows information of different APIs.

Reply:
Alright let's chat to @sqren about it later.